scripts/karma.conf.js

```diff
@@ -14,18 +14,19 @@ module.exports = function (config) {
     frameworks: ['jasmine'],
 
     // list of files / patterns to load in the browser
-    files: [{
-      pattern: './karma-shim.js',
-      watched: false
-    }],
+    files: [
+      { pattern: './karma-shim.js', watched: false },
+      { pattern: './../src/themes/default.theme.scss', watched: false }
```

Comment:
Do we need to include theme for unit tests? It would be better if all unit tests are working with core styles only.